Source/Scene/ParticleSystem.js

```diff
@@ -817,7 +1009,25 @@ define([
     };
 
     /**
-     * A function used to apply a force to the particle on each time step.
+     * A function used to modify attributes of the particle at each time step. This can include force modifications,
+     * color, sizing, etc.
```

Comment:
This can include force modifications, color, sizing, etc.

This is good clarification, but I would remove it and perhaps add a link to the particle tutorial instead for more clarification.